lib/pwinty/order.rb

```diff
@@ -17,6 +17,7 @@ class Order < Pwinty::Base
         attribute :stateOrCounty, Types::String.optional
         attribute :status, Types::String
         attribute :payment, Types::String
+        attribute :packingSlipUrl, Types::String.optional
```

Comment:
The `packingSlipUrl` is often absent from the response, which will cause an error to be raised when we declare that it should be present.

The suggestion below will allow it to be passed in - fixing the error you see - while also allowing it to be absent.

```suggestion
        attribute? :packingSlipUrl, Types::String.optional
```

If you implement this suggestion, you'll notice that the tests will pass and I will be happy to merge it in.

Reply:
Rgr, will do it now